src/components/careers/Positions/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+import styled from 'styled-components'
+
+import { Card } from './Card'
+import ContentWrapper from '../../Layout/ContentWrapper'
+import { SectionTitle } from '../styled'
+import { sizes } from '../../../styles/styleUtils'
+
+const Text = styled.p`
+  text-align: center;
+  font-size: 24px;
+  line-height: 32px;
+  max-width: 730px;
+  margin: 0 auto;
+
+  & strong {
+    font-weight: bold;
+  }
+`
+
+type JobMeta = {
+  id: number
+  name: string
+  value: unknown
+  value_type: string
+}
+
+type Job = {
+  absolute_url: string
+  data_compliance: unknown[]
+  id: number
+  internal_job_id: number
+  location: {
+    name: string
+  }
+  metadata: JobMeta[]
+  requisition_id: unknown
```

Comment:
Seems like this is currently never used.

```suggestion
  requisition_id: null
```